form-data/form-data.d.ts

```diff
@@ -1,16 +1,31 @@
 // Type definitions for form-data
 // Project: https://github.com/felixge/node-form-data
-// Definitions by: Carlos Ballesteros Velasco <https://github.com/soywiz>
+// Definitions by: Carlos Ballesteros Velasco <https://github.com/soywiz>, Leon Yu <https://github.com/leonyu>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 // Imported from: https://github.com/soywiz/typescript-node-definitions/form-data.d.ts
 
+declare namespace NodeFormData {
```

Comment:
Yea, I was using global type to avoid the ES6 `not a module` error. Because I did global, it was colliding with `FormData` from `dom.d.ts`, so I renamed it. 

Your solution is perfect. I didn't know creating a namespace also solves the ES6 `no a module` error